CHANGELOG.md

```diff
@@ -98,12 +87,15 @@ update the database cluster to use the new configuration.
     unexpected insertion failure on PATCH.
 - **CUMULUS-3181**
   - Fixed `sqsMessageRemover` lambda to correctly retrieve ENABLED sqs rules.
-- The `getLambdaAliases` function has been removed from the `@cumulus/integration-tests` package
-- The `getLambdaVersions` function has been removed from the `@cumulus/integration-tests` package
 - **CUMULUS-3189**
   - Upgraded `cumulus-process` and `cumulus-message-adapter-python` versions to
     support pip 23.0
 - README shell snippets better support copying
+- **CUMULUS-3140**
+  - Update release note to include cumulus-api release
+- **CUMULUS-3193**
+  - Update eslint config to better support typing
```

Comment:
These two belong to ### Changed.

Reply:
Hm great catch! https://github.com/nasa/cumulus/pull/3282/commits/9e19729c3289927bf5665174e14cc7e845e6274a